modelmapper/etl.py

```diff
@@ -49,7 +49,7 @@ def report_exception(self, e):
         raise NotImplementedError('Please implement this method in your subclass.')
 
     def get_hash_of_bytes(self, item):
-        return mmh3.hash(item)
+        return mmh3.hash64(item, False)[0]
```

Comment:
2nd parameter is a seed value, 3rd parameter says to only use the x86 implementation (or rather, not optimize for x64). Not sure how the hash64 method is handling this, but it should probably be something like:

`return mm3.hash64(item, SIGNATURE_SEED, False)[0]`

and SIGNATURE_SEED is just some number you set up (example on their site uses 42... don't use 42)

Reply:
just used the key word arg so I won't have to specify a seed